addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -209,13 +209,13 @@ pscanrules.insecureauthentication.refs = https://cheatsheetseries.owasp.org/chea
 pscanrules.insecureauthentication.soln = Protect the connection using HTTPS or use a stronger authentication mechanism
 
 pscanrules.insecureformload.desc = This check looks for insecure HTTP pages that host HTTPS forms. The issue is that an insecure HTTP page can easily be hijacked through MITM and the secure HTTPS form can be replaced or spoofed.
-pscanrules.insecureformload.extrainfo = The response to the following request over HTTP included an HTTPS form tag action attribute value:\r\n\r\n{0}The context was:\r\n\r\n{1}
+pscanrules.insecureformload.extrainfo = The response to the following request over HTTP included an HTTPS form tag action attribute value:\r\n\r\n{0}\r\nThe context was:\r\n\r\n{1}
 pscanrules.insecureformload.name = HTTP to HTTPS Insecure Transition in Form Post
 pscanrules.insecureformload.refs = 
 pscanrules.insecureformload.soln = Use HTTPS for landing pages that host secure forms.
 
 pscanrules.insecureformpost.desc = This check identifies secure HTTPS pages that host insecure HTTP forms. The issue is that a secure page is transitioning to an insecure page when data is uploaded through a form. The user may think they're submitting data to a secure page when in fact they are not.
-pscanrules.insecureformpost.extrainfo = The response to the following request over HTTPS included an HTTP form tag action attribute value:\r\n\r\n{0}The context was:\r\n\r\n{1}
+pscanrules.insecureformpost.extrainfo = The response to the following request over HTTPS included an HTTP form tag action attribute value:\r\n\r\n{0}\r\nThe context was:\r\n\r\n{1}
```

Comment:
For a separate PR should this simply use \n ?
Should that be enforced by the formatter?

Reply:
+1 to \n.